DevelopersHand/src/application/Player.java

```diff
@@ -0,0 +1,93 @@
+package application;
+
+import java.util.*;
+
+public class Player {
+  
+  private String name;
+  private ArrayList<String> skills;
+  //private ArrayList<Card> deck;
+  private String objective;
+  private int XP;
+  private int bugs;
+  
+  public Player(String name){
+	this.setName(name);
```

Comment:
For constructors, you don't need to use the setX() method.
In this case, just do `this.name = name`.